flutter-idea/src/io/flutter/run/daemon/DaemonConsoleView.java

```diff
@@ -67,6 +72,9 @@ public void print(@NotNull String text, @NotNull ConsoleViewContentType contentT
     if (contentType != ConsoleViewContentType.NORMAL_OUTPUT) {
       writeAvailableLines();
 
+      if (sendErrorsToAnalytics && text.length() > 0) {
+        analytics.sendEvent("potential-disconnect", text);
```

Comment:
add the 
WorkspaceCache.getInstance(getProject()).isBazel()
check right here and note we are only logging for internal users and this logging needs to not be enabled for all users without being modified to only test for matches for specific known disconnect strings.